backend/command/src/shared/functions/get-slug.ts

```diff
@@ -1,6 +1,6 @@
 import slugify from 'slugify';
 
-export const getSlug = (str: string, space: string) => {
+export const getSlug = (str: string, space = '-') => {
```

Comment:
Add comment/jsdoc that explains the default behaviour of the function. It should be visible at first glance that the function defaults to '-'

Reply:
did